charts/blocky/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "v0.8"
 description: DNS proxy as ad-blocker for local network
 name: blocky
-version: 3.2.0
+version: 3.2.1
```

Comment:
With this being a new feature, do you mind bumping the chart to 3.3.0?

Reply:
Done!